python_modules/jupedsim/tests/test_distributions.py

```diff
@@ -226,66 +233,31 @@ def test_distribution_in_circle_by_density_creates_correct_amount():
     assert len(samples) == 78
 
 
-def test_catch_wrong_inputs_for_wrong_polygon_type():
-    polygon = [(0, 0), (10, 0), (10, 10), (0, 10)]
-    # polygon is no Shapely Polygon
-    center_point, circle_segment_radii, numbers_of_agents = (0, 0), [(1, 3)], [5]
-    with pytest.raises(distributions.IncorrectParameterError):
-        distributions.__catch_wrong_inputs(polygon=polygon, center_point=center_point,
-                                           circle_segment_radii=circle_segment_radii, fill_parameters=numbers_of_agents)
-
+test_parameters = [
+    [[(0, 0), (10, 0), (10, 10), (0, 10)], 5, [(1, 3)], [5], distributions.IncorrectParameterError],
+    [[(0, 0), (10, 0), (10, 10), (0, 10)], [0], [(1, 3)], [5], distributions.IncorrectParameterError],
+    [[(0, 0), (10, 0), (10, 10), (0, 10)], (0, 0), [(1, 3), (3, 5)], [5], distributions.IncorrectParameterError],
+    [[(0, 0), (10, 0), (10, 10), (0, 10)], (0, 0), [(-1, 3), (3, 5)], [5, 5], distributions.NegativeValueError],
+    [[(0, 0), (10, 0), (10, 10), (0, 10)], (0, 0), [(1, 3), (2, 5)], [5, 5], distributions.OverlappingCirclesError],
+    [[(0, 0), (10, 0), (10, 10), (0, 10)], (0, 0), [(0, 2), (3, 5), (2, 4)], [5, 5, 5],
+     distributions.OverlappingCirclesError],
+    [[(0, 0), (10, 0), (10, 10), (0, 10)], (0, 0), [(0, 5), (8, 3)], [5, 5], distributions.OverlappingCirclesError]
+]
```

Comment:
There should be one (parametrized) test for each type of error. This makes it easier to understand which error has occurred during testing.
For this example there should be 3 test methods.